.github/workflows/win.yml

```diff
@@ -14,5 +14,22 @@ jobs:
       - name: Check out code into the Go module directory
         uses: actions/checkout@v2
 
-      - name: build
-        run: go build
\ No newline at end of file
+      - uses: actions/cache@v2
+        with:
+          path: |
+            ~/go/pkg/mod              # Module download cache
+          key: ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}
+          restore-keys: |
+            ${{ runner.os }}-go-
+            
+      - name: Build
+        run: go build
+        
+      - name: Test
+        run: make -- --ci-test
```

Comment:
Rather than adding a new specific make target just for windows, I think we could just write `go test -timeout 2m -race ./...` here in the action file